src/components/menu/menu.ts

```diff
@@ -0,0 +1,105 @@
+import Block from 'core/Block';
+import './menu.css';
+import { MenuProps } from './types';
+import { Popup } from 'utils/classes/Popup';
+import { config } from 'utils/consts';
+import plus from 'img/plus.svg';
+import close from 'img/close.svg';
+import photo from 'img/photo.svg';
+import file from 'img/file.svg';
+import location from 'img/location.svg';
+
+export class Menu extends Block {
+  static componentName = 'Menu';
+  constructor({ isUser }: MenuProps) {
+    super({ isUser });
+  }
+  protected getStateFromProps(props: MenuProps): void {
+    this.state = {
+      isUser: props.isUser,
+
+      handleAddUserPopup: () => {
+        new Popup(
+          config.popupAddUserSelector,
+          config.menuBtnAddUserSelector,
+          config.isOpenPopupSelecot,
```

Comment:
опечатка по всей видимости
```suggestion
          config.isOpenPopupSelector,
```